dali/base/dasds.cpp

```diff
@@ -5236,7 +5236,7 @@ class CStoreHelper : public CInterface, implements IStoreHelper
                 MilliSleep(100);
             }
         }
-        ~CheckDeltaBlock()
+        ~CheckDeltaBlock() noexcept(false)
```

Comment:
Should we not instead remove the throw in the destructor and catch/keep the exception.
Then ensure a member function is called to throw it?
(There are a couple of instances of CheckDeltaBlock, would probably need to ensure that the scope they're, is in a try/catch block and call after)